### PR TITLE
About update

### PR DESCRIPTION
**Pull request by shamshirz**
Check it [Here](https://sylver-studios-pr-32.herokuapp.com/about/)

➕/ ➖   Image loads in twitter card (Pretty sure it will work once deployed)
✅ Add Description for twitter

Adding images to pretty the page up, and twitter info so the post looks nice.

### CODE REVIEW

File: assets/css/style.scss [hunk 1]
```diff
@@ -33,3 +33,21 @@
...
+  justify-content: space-evenly;
+  align-items: center;
+
+  blockquote {
+    height: fit-content;
+  }
+}
+
+.about-image {
+  height: 200px;
```
Comment (rjdellecese) [inline assets/css/style.scss:48]: This looks bad on mobile :( Can we maintain the aspect ratio? I think that would solve it...

File: aaron.md [hunk 2]
```diff
@@ -3,10 +3,14 @@ layout: page
 title: Aaron
 permalink: /about/aaron/
 youtubeId: 41dsGsrmCrI
+author_twitter: shamshirz
+image: /assets/shamshirz.png
```
Comment (rjdellecese) [inline aaron.md:7]: So this is for Twitter somehow, right?

File: aaron.md [hunk 3]
```diff
@@ -3,10 +3,14 @@ layout: page
...
 ---
 
 Aaron is an adventurer of all things software. He came to Elixir from a Java and Node background to fall in love with the community and the functional coding style. Currently working for [Corvus Insurance](https://www.corvusinsurance.com/), Aaron writes primarily in Elm and Elixir while experimenting with game development in his spare time. Often found in his natural habitat of soccer fields located near coffee shops.
 
+![Aaron themed player card with social links]({{ "/assets/shamshirz.png" | relative_url }}){:class="aaron-page-image"}
```
Comment (rjdellecese) [inline aaron.md:12]: This also looked weird for me on mobile

**Review by rjdellecese — COMMENTED** [review]
As noted in the comments, it looked a little weird on mobile for me (I just happened to view it on my phone initially, which is how I found that out), but that doesn't need to prevent this from being merged.

**Comment by shamshirz** [comment]
Aspect Ratio repaired. Slightly weird looking, but big improvement. Merging wildly now
MobileAspectRatios

**Pull request merged**